hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/ozoneimpl/AbstractContainerScannerMetrics.java

```diff
@@ -42,6 +43,8 @@ public abstract class AbstractContainerScannerMetrics {
   private MutableGaugeInt numUnHealthyContainers;
   @Metric("number of iterations of scanner completed since the restart")
   private MutableCounterInt numScanIterations;
+  @Metric("total time that the container scanned has been running, in milliseconds.")
+  private MutableCounterLong totalRunTimes;
```

Comment:
We can rename the getters and setters too.

Reply:
I see how `totalRunTimes` might be used to imply that this is the sum of times of all the runs, but I think `totalRunTime` still conveys this since it is the "total".